src/main/resources/config.properties

```diff
@@ -1,4 +1,5 @@
 magento.url=https://magento.softwaretestingboard.com/
 magento.logout =https://magento.softwaretestingboard.com/customer/account/logoutSuccess/
 user.magento.username=irinakorobeynik@coherentsolutions.com
-user.magento.password=q1W23e4567Rty!
\ No newline at end of file
+user.magento.password=q1W23e4567Rty!
+path.for.screenshots=src/main/resources
```

Comment:
let's update it to target/atrifacts/screenshots
resources are external data storages, screenshots are artifacts